sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlDialectSpecTest.java

```diff
@@ -2174,6 +2175,246 @@ public void testZetaSQLNestedQueryFive() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  /////////////////////////////////////////////////////////////////////////////
+  // NUMERIC type tests
+  /////////////////////////////////////////////////////////////////////////////
+
+  @Test
+  public void testNumericLiteral() {
+    String sql =
+        "SELECT NUMERIC '0', "
+            + "NUMERIC '123456', "
+            + "NUMERIC '-3.14', "
+            + "NUMERIC '-0.54321', "
+            + "NUMERIC '1.23456e05', "
+            + "NUMERIC '-9.876e-3', "
+            + "NUMERIC '-99999999999999999999999999999.999999999', "
```

Comment:
Add comment on these are the min/max values that can be represented as ZetaSQL NUMERIC?

Reply:
OK. Done.